roles/content_view_version_cleanup/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: find all CVs
+  theforeman.foreman.resource_info:
+    server_url: "{{ server_url | default(omit) }}"
+    username: "{{ username | default(omit) }}"
+    password: "{{ password | default(omit) }}"
+    validate_certs: "{{ validate_certs | default(omit) }}"
+    organization: "{{ organization }}"
+    resource: content_views
+  register: raw_cvs
+
+- name: delete ccv versions
+  include_tasks: delete_cv_versions.yml
+  vars:
+    cv_name: "{{ ccv.name }}"
+    cv_versions: "{{ ccv | json_query(content_view_version_cleanup_versions_filter) }}"
+  loop: "{{ raw_cvs | json_query('resources[?composite]') }}"
+  loop_control:
+    label: "{{ ccv.label }}"
+    loop_var: ccv
+  when: ccv | json_query(content_view_version_cleanup_versions_filter)
```

Comment:
should `jmespath` be in dependencies?

Reply:
aka requirements

Reply:
Fair, I think it's only a weak dep of Ansible.